src/main/java/core/basesyntax/service/impl/operation/BalanceOperation.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.service.impl.operation;
+
+import core.basesyntax.service.OperationHandler;
+
+public class BalanceOperation implements OperationHandler {
```

Comment:
The class name `BalanceOperation` is good, but ensure that all class names follow the Java naming convention and are descriptive of their purpose.

Reply:
++